packages/snaps-utils/src/snaps.ts

```diff
@@ -1,8 +1,8 @@
 import { BlockReason } from '@metamask/snaps-registry';
 import { assert, Json, SemVerVersion } from '@metamask/utils';
-import { sha256 } from '@noble/hashes/sha256';
 import { base64 } from '@scure/base';
 import { SerializedEthereumRpcError } from 'eth-rpc-errors/dist/classes';
+import stringify from 'fast-json-stable-stringify';
```

Comment:
More descriptive name might be helpful
```suggestion
import stableStringify from 'fast-json-stable-stringify';
```